lib/rules/prefer-numeric-literals.js

```diff
@@ -6,13 +6,42 @@
 "use strict";
 
 //------------------------------------------------------------------------------
+// Helpers
+//------------------------------------------------------------------------------
+
+/**
+ * Checks to see if a CallExpression's callee node is `parseInt` or
+ * `Number.parseInt`.
+ * @param {ASTNode} calleeNode The callee node to evaluate.
+ * @returns {boolean} True if the callee is `parseInt` or `Number.parseInt`,
+ * false otherwise.
+ */
+function isParseInt(calleeNode) {
+    if (calleeNode) {
```

Comment:
Okay, I can remove it. Maybe you should switch your review to Request Changes so nobody accidentally merges this in?